src/ticket/ticket.module.ts

```diff
@@ -0,0 +1,13 @@
+import {Module} from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import {Ticket} from './ticket.entity';
+import {TicketService} from './ticket.service';
+import { TicketRepository } from './ticket.repository';
+
+@Module({
+    imports: [TypeOrmModule.forFeature([Ticket])],
+    providers: [TicketService, TicketRepository],
+    exports: [TicketService],
+})
+export class TicketModule{}
+
```

Comment:
module should have functionality for issuing tickets, listing tickets, scanning ticket - depending on logged-in role; we thus need to add roles first